packages/components/src/timeline/timeline-item.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import GridIcon from 'gridicons';
+
+const GRIDICON_SIZE = 16;
```

Comment:
>they are also off-center
That's interesting! They're centered in Storybook 🤔. I'd be interested in seeing what's happening differently in wcpay.

I'll reach out in Slack so we can chat about this in more detail.

>18, since the icons are pretty small

No problem! This seems bigger in Storybook, so again I'd ben interested in seeing what's different in WCPay.

Reply:
Screenshot:

![image](https://user-images.githubusercontent.com/13835680/79858484-12e4b180-8402-11ea-9224-03a1a1c36c18.png)

Reply:
This has been resolved, both as a discussion in p1587567205003600-slack-wcpay-events and with some stricter CSS.